source/jormungandr/jormungandr/instance.py

```diff
@@ -219,6 +219,13 @@ def autocomplete(self):
             )
         return backend
 
+    @property
+    def kraken_autocomplete(self):
+        backend = global_autocomplete.get('kraken')
```

Comment:
Can't we loop over available autocomplete instead of looking for a key?
And maybe rename method `is_possible_fallback()`, changing a bit the logic to avoid requesting mimir when kraken fails? As we don't check that the requested id starts with `stop_point:` it seems doable.

Reply:
I tried another implementation based on what you asked.

I'm not sure the abstraction helps the understanding of the code, but I'll let you judge it